templates/shared_services/sonatype-nexus-vm/terraform/vm.tf

```diff
@@ -112,7 +113,7 @@ resource "azurerm_linux_virtual_machine" "nexus" {
   source_image_reference {
     publisher = "Canonical"
     offer     = "0001-com-ubuntu-server-jammy"
-    sku       = "22_04-lts-gen2"
+    sku       = "22_04-lts"
```

Comment:
Why would we not use a Gen2 image here?

Reply:
think it got overwritten on a merge, I've just it back.